README.md

```diff
@@ -25,6 +25,12 @@ If you wish to [use server-side rendering (SSR)](https://docs.astro.build/en/gui
 
 Add the Deno adapter to enable SSR in your Astro project with the following steps:
 
+1. Update or create `.npmrc` in your project with:
+
+   ```text
+   @jsr:registry=https://npm.jsr.io
+   ```
```

Comment:
This feels like an unnecessary friction to the users. Can we use other deep merge package which is available in npm?

Reply:
@kt3k  I switched to lodash.

Reply:
Nice. Thanks!